Config/Sample.cfg

```diff
@@ -117,6 +117,7 @@ RegisteredUserData              : 0             # SEI message (0= OFF, 1=ON )
 UnregisteredUserData            : 0             # SEI message (0= OFF, 1=ON )
 RecoveryPoint                   : 0             # SEI message (0= OFF, 1=ON )
 TemporalId                      : 1             # 0: OFF, 1: Insert temporal ID in NAL units 
+SwitchThreadsToRtPriority       : 0             # 0: OFF, 1: Switch threads to real time priority 
```

Comment:
This change will make a great latency difference. Do you want to keep "SwitchThreadsToRtPriority" default to 1?

Reply:
Default has be set to 1